lib/lita/handlers/reviewme.rb

```diff
@@ -96,6 +96,7 @@ def comment_on_github(response, room: get_room(response))
         rescue Octokit::Error
           response.reply("Unable to check who issued the pull request. Sorry if you end up being assigned your own PR!")
         end
+        return response.reply('Sorry, no reviewers found') unless reviewer
```

Comment:
Makes sense. It does seem like this is due some refactoring, but I suppose we'll just end up waiting for the next opportunity 😄